components/usage/pkg/apiv1/usage_test.go

```diff
@@ -23,7 +23,16 @@ import (
 	"gorm.io/gorm"
 )
 
-func TestUsageService_ReconcileUsage(t *testing.T) {
+// TestRunAllUsageApiTests runs all tests sequentially to avoid race conditions on the DB
+func TestRunAllUsageApiTests(t *testing.T) {
```

Comment:
I genuinely have no idea why this would work. Tests in a given package are run sequentially, tests across packages are run in parallel. So this should have exactly 0 effect

Reply:
Uh, my bad, it was too late :see_no_evil: 

Then the cause here is somewhere else, maybe different _test.go units accessing the same DB tables...?